Atomic/diff.py

```diff
@@ -26,9 +31,13 @@ def cli(subparser):
     diffp.add_argument("-v", "--verbose", default=False, action='store_true',
                        help=_("Show verbose output, listing all RPMs"))
 
+
 class Diff(Atomic):
 
     def diff_tty(self):
+        if self.args.no_files and not self.args.metadata and not self.args.rpms:
+            raise ValueError("When repressing a file diff, you must choose to diff RPMs (-r) or metadata (-m)")
```

Comment:
@rhatdan I dont know, i struggled with this.  On one hand it would be better to have them choose one or more from files, -m, -r.  But then the notion of a default isnt there and I really want to have the command work out of the box.  So if someone suppresses file comparison, what is the logical default.  I came to the conclusion that there isnt one.  Input welcome.